memberships/views.py

```diff
@@ -179,6 +179,13 @@ def stripe_webhook(request):
             json.loads(request.body), settings.STRIPE_SECRET_KEY
         )
         stripe_webhook = StripeWebhook()
+        if event["type"] == "invoice.payment_failed":
+            f_payment = FailedPayment.create(
```

Comment:
Ah, this will now also need to be changed.

```suggestion
            f_payment = FailedPayment.objects.create(
```